bin/configs/typescript-fetch-sagas-and-records.yaml

```diff
@@ -0,0 +1,16 @@
+generatorName: typescript-fetch
+outputDir: samples/client/petstore/typescript-fetch/builds/sagas-and-records
+inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing-saga-and-records.yaml
```

Comment:
Can we use OpenAPI 3.0 spec instead as we're doing the same for other samples from openapi 2.0 to 3.0 spec?

Reply:
If this can wait, I would prefer to take a look at this at a later time. I am not sure of the implications of changing this. It is probably very easy, but I am not sure. Would prefer to proceeed with a first merge and upgrade in a next phase. See my additional comments below regarding the futur plan for this upgrade.